tools/winmd2objc/include/WinRTSymbolTable.h

```diff
@@ -34,7 +34,7 @@ class SymbolNode;
 //  intentional - if we ever hit the templated version, it's an error.
 template <class T>
 bool CompareStringLess(T left, T right) {
-    NT_ASSERT(FALSE);
+    assert(FALSE);
```

Comment:
NT_ASSERT*
Also NT_ASSERT is #define NT_ASSERT assert // @_@

---
In reply to: [117041110](https://github.com/Microsoft/WinObjC/pull/2708#discussion_r117041110) [](ancestors = 117041110)